blueprints/serverless/cloud-run-microservices/variables.tf

```diff
@@ -0,0 +1,78 @@
+/**
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+variable "custom_domain" {
+  description = "Custom domain for the Load Balancer."
+  type        = string
+  default     = "service-b.acme.org"
+}
+
+variable "ip_ranges" {
+  description = "IP ranges or IPs used by the VPC."
+  type        = map(string)
+  default = {
+    subnet_main       = "10.0.1.0/24"
+    subnet_proxy      = "10.10.0.0/24"
+    subnet_vpc_access = "10.10.10.0/28"
+    subnet_vpc_direct = "10.8.0.0/26"
+    psc_addr          = "10.0.0.100"
+  }
+}
+
+variable "main_project" {
```

Comment:
can we reafctor this so we have a single top-level `project_configs` variable with optional members for `main` and `service`?

Reply:
Done